com.unity.ml-agents/Runtime/Demonstrations/DemonstrationSummary.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using MLAgents.Policies;
+
+namespace MLAgents.Demonstrations
+{
+    /// <summary>
+    /// Summary of a loaded Demonstration file. Only used for display in the Inspector.
+    /// </summary>
+    [Serializable]
+    internal class DemonstrationSummary : ScriptableObject
+    {
+        public DemonstrationMetaData metaData;
```

Comment:
Does the DemonstrationSummary need DemonstrationMetaData ? I thought the two were side by side anyway.

Reply:
The fields from the DemonstrationMetaData are displayed in the inspector. DemonstrationSummary is _only_ used by the DemonstrationDrawer, it's not a full Demonstration (that's why I renamed it).